app/scripts/controllers/external-transactions.js

```diff
@@ -282,7 +281,10 @@ export default class IncomingTransactionsController {
         value: bnToHex(new BN(etherscanTransaction.value)),
       },
       hash: etherscanTransaction.hash,
-      type: TRANSACTION_TYPES.INCOMING,
+      type:
```

Comment:
The type parameter here will not be this simple, this could be any currently supported type of transaction not just sent or incoming. In the main transaction controller we use a method that determines what the transaction's type is.. we could potentially reuse some of that logic.